sdks/python/apache_beam/utils/dependency.py

```diff
@@ -423,7 +423,62 @@ def _stage_dataflow_sdk_tarball(sdk_remote_location, staged_path, temp_dir):
         'Staging Dataflow SDK tarball from %s to %s',
         sdk_remote_location, staged_path)
     _dependency_file_copy(sdk_remote_location, staged_path)
+  elif sdk_remote_location == 'pypi':
+    logging.info('Staging the SDK tarball from PyPI to %s', staged_path)
+    _dependency_file_copy(_download_pypi_sdk_package(temp_dir), staged_path)
   else:
     raise RuntimeError(
         'The --sdk_location option was used with an unsupported '
         'type of location: %s' % sdk_remote_location)
+
+
+def get_required_container_version():
+  """Returns the Google Cloud Dataflow container version for remote execution.
+
+  Raises:
+    pkg_resources.DistributionNotFound: if one of the expected package names
+      are not found: 'google-cloud-dataflow' (right now) and 'apache-beam'
+      (in the future).
+  """
+  # TODO(silviuc): Handle apache-beam versions when we have official releases.
+  import pkg_resources as pkg
+  try:
+    version = pkg.get_distribution('google-cloud-dataflow').version
```

Comment:
Can we define PACKAGE_NAME = 'google-cloud-dataflow' at an outer scope and reuse it?